src/main/scala/io/sdkman/changelogs/BldMigration.scala

```diff
@@ -0,0 +1,21 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "077")
+class AntMigration {
+  @ChangeSet(
+    order = "001",
+    id = "001-add_bld_candidate",
+    author = "gbevin"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "bld",
+      name = "Bld",
+      description = "bld is a new build system that allows you to write your build logic in pure Java. bld was created because we're not really interested in build tools. We use them because we have to, but we'd rather just get on with coding the real stuff.",
```

Comment:
I love this blurb! Can you drop the word `new`, please? 6 months from now, it will no longer be new :laughing: